packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1841,7 +1842,7 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
       _CreationLocation location = knownLocations[id]!;
       expect(location.file, equals(file));
       // ClockText widget.
-      expect(location.line, equals(53));
+      expect(location.line, equals(54));
```

Comment:
How does `--precompiled` affect the results of non-Web tests?

Reply:
it didn't, the test depends on exact line numbers which changed when i added the annotation

Reply:
Gotcha